src/main/windows/mainWindow.ts

```diff
@@ -324,6 +324,10 @@ export class MainWindow extends EventEmitter {
     private onFocus = () => {
         // Only add shortcuts when window is in focus
         if (process.platform === 'linux') {
+            const kde = isKDE();
+            if ((!this.win || this.win.isMinimized()) && kde) {
+                return;
+            }
             globalShortcut.registerAll(ALT_MENU_KEYS, () => {
```

Comment:
So this doesn't break anything now, but there's going to be an incoming PR allowing users to use the old native style window borders, which include the old menus at the top.

There's a bug in Electron that brings those back when you press the ALT key even if you've hidden them. I just tested and it doesn't break under the current setup, but it will with that incoming change.

Just to be safe, can we run that check underneath this call so that it still happens for Linux?